test/profiles/advanced/hubble/cilium/values.yaml

```diff
@@ -0,0 +1,3 @@
+enabledPlugin_linux: '["dns", "ciliumeventobserver"]'
```

Comment:
for non go code, would it be possible to add a go-esque registry using embed fs that we can determine if these files exist at runtime? check this out:

https://github.com/Azure/azure-container-networking/blob/master/test/e2e/manifests/manifests_test.go

that way we know values.yaml exists in the path we expect, and less prone for e2e's failing 20 minutes in because a yaml/manifest is missing or in a wrong directory